src/legacy/core_plugins/table_vis/public/agg_table/__tests__/_table.js

```diff
@@ -353,6 +354,60 @@ describe('AggTable Directive', function () {
     });
   });
 
+  it('renders percentage columns', async function () {
+    $scope.dimensions = {
+      buckets: [
+        { accessor: 0, params: {} },
+        { accessor: 1, format: { id: 'date', params: { pattern: 'YYYY-MM-DD' } } },
+      ],
+      metrics: [
+        { accessor: 2, format: { id: 'number' } },
+        { accessor: 3, format: { id: 'date' } },
+        { accessor: 4, format: { id: 'number' } },
+        { accessor: 5, format: { id: 'number' } },
+      ],
+    };
+    const response = await tableAggResponse(
+      tabifiedData.oneTermOneHistogramBucketWithTwoMetricsOneTopHitOneDerivative,
+      $scope.dimensions
+    );
+    $scope.table = response.tables[0];
+    $scope.percentageCol = 'Average bytes';
+
+    const $el = $(`<kbn-agg-table
+      table="table"
+      dimensions="dimensions"
+      percentage-col="percentageCol"
+    ></kbn-agg-table>`);
+
+    $compile($el)($scope);
+    $scope.$digest();
+
+    const $headings = $el.find('th');
+    expect($headings.length).to.be(7);
+    expect(
+      $headings
+        .eq(3)
+        .text()
+        .trim()
+    ).to.be('Average bytes percentages');
+
+    const countColId = $scope.table.columns.find(col => col.name === $scope.percentageCol).id;
+    const counts = $scope.table.rows.map(row => row[countColId]);
+    const total = counts.reduce((sum, curr) => sum + curr, 0);
+    const $percentageColValues = $el.find('tbody tr').map((i, el) =>
+      $(el)
+        .find('td')
+        .eq(3)
+        .text()
+    );
+
+    $percentageColValues.each((i, value) => {
+      const percentage = `${_.round((counts[i] / total) * 100, 1)}%`;
```

Comment:
Can we use `Math.round` instead? I think it's the default rounding function used by `@elastic/numeral` if I'm not wrong

Reply:
`Math.round` doesn't support a precision param so it's a bit more work... 

Leaving it as lodash for now but changing the import to be more specific to only import `round`.